Donation-platform/src/main/java/com/google/charities/AddCharitiesFromJSON.java

```diff
@@ -0,0 +1,165 @@
+  private static final String ANIMAL_IMAGE = "https://i.pinimg.com/originals/51/31/0d/51310de57fc8f015e4020ed258c763ae.jpg";
+  private static final String ARTS_IMAGE = "https://tradinginsider.fr/wp-content/uploads/2018/06/culture-768x473.png";
+  private static final String EDU_IMAGE = "https://blogs.edweek.org/edweek/inside-school-research/Hand-Raised-Young-Boy-Classroom-GETTY-Blog-560x292.jpg";
+  private static final String ENV_IMAGE = "https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/nature-quotes-1557340276.jpg";
+  private static final String HEALTH_IMAGE = "https://www.debt.com/wp-content/uploads/2018/06/Doctor-helps-patient.jpg";
+  private static final String HUMAN_IMAGE = "https://arc.losrios.edu/arc/main/img/instruction/ARC-BSS-Division/ARC-Human-Services/ARC-human-services-940x529.jpg";
+  private static final String INTL_IMAGE = "http://opiniojuris.org/wp-content/uploads/63.jpg";
+  private static final String RIGHTS_IMAGE = "https://media2.s-nbcnews.com/i/newscms/2020_23/3382051/200606-washington-demonstration-al-1549_e1c57b283c615f899ba66a0168460c8b.jpg";
+  private static final String RELI_IMAGE = "https://www.sbs.com.au/topics/sites/sbs.com.au.topics/files/styles/full/public/gettyimages-496701302.jpg";
+  private static final String COMM_IMAGE = "https://www.wxriskglobal.com/wp-content/uploads/2018/03/shutterstock_660487873.png";
+  private static final String RESEARCH_IMAGE = "https://www.apa.org/images/title-research-publishing_tcm7-251846.jpg";
+  
+  // LIST OF TAG NAMES
+  private static final List<String> tags = new ArrayList<String>(Arrays.asList(ANIMAL, ARTS, EDU, ENV, HEALTH, HUMAN, INTL, RIGHTS, RELI, COMM, RESEARCH)); 
+ 
+  // HARDCODED MAPPING OF TAGS TO TRENDING SCORES
+  public static final Map<String, Integer> tagScores =
+    new HashMap<String, Integer>() {
+        {
+          put(ANIMAL, 15);
+          put(ARTS, 10);
+          put(EDU, 30);
+          put(ENV, 30);
+          put(HEALTH, 50);
+          put(HUMAN, 40);
+          put(INTL, 30);
+          put(RIGHTS, 50);
+          put(RELI, 20);
+          put(COMM, 40);
+          put(RESEARCH, 20);
+        }
+    };
+ 
+  // MAPPING OF TAGS TO IMAGES
+  // TODO: figure out why tag images fail to display on browse page
+  public static final Map<String, String> tagImages =
+    new HashMap<String, String>() {
+        {
+          put(ANIMAL, ANIMAL_IMAGE);
+          put(ARTS, ARTS_IMAGE);
+          put(EDU, EDU_IMAGE);
+          put(ENV, ENV_IMAGE);
+          put(HEALTH, HEALTH_IMAGE);
+          put(HUMAN, HUMAN_IMAGE);
+          put(INTL, INTL_IMAGE);
+          put(RIGHTS, RIGHTS_IMAGE);
+          put(RELI, RELI_IMAGE);
+          put(COMM, COMM_IMAGE);
+          put(RESEARCH, RESEARCH_IMAGE);
+        }
+    };
+ 
+  public AddCharitiesFromJSON(DatastoreService ds, DbCalls db) {
+    this.ds = ds;
+    this.db = db;
+  }
+ 
+  public DbCalls getDbCalls() {
+    return this.db;
+  }
+ 
+  // Add hardcoded tags to DB with scores
```

Comment:
why are we doing this

Reply:
addTags allows for the 11 tags we're using to be added to the db if they're not already in the db.